spec/controllers/wishlists_controller_spec.rb

```diff
@@ -1,83 +1,123 @@
 require "rails_helper"
 
 RSpec.describe WishlistsController do
-  let(:valid_parameters) { attributes_for(:wishlist, user_id: create(:user).id) }
+  context "when user is logged in" do
+    sign_in_user
```

Comment:
It's a method that gets extended into each controller spec and invokes a `before` block defined in it. But it's a not obvious behavior, so I've updated it to be included and invoked on normal terms in explicit `before` blocks.